heft-plugins/heft-localization-typings-plugin/src/schemas/options.schema.json

```diff
@@ -0,0 +1,58 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object",
+  "additionalProperties": false,
+  "properties": {
+    "exportAsDefaultOptions": {
+      "oneOf": [
+        {
+          "type": "boolean"
+        },
+        {
+          "type": "object",
+          "additionalProperties": false,
+          "properties": {
+            "interfaceName": {
+              "type": "string",
+              "description": "The interface name for the default wrapped export. Defaults to \"IExport\""
+            },
+            "documentationComment": {
+              "type": "string",
+              "description": "This value is placed in a documentation comment for the exported default interface."
+            },
+            "inferInterfaceNameFromFilename": {
+              "type": "boolean",
+              "description": "When set to true, the default export interface name will be inferred from the filename. This takes precedence over the interfaceName option."
+            }
```

Comment:
Should probably update the schema to only allow one of `inferInterFaceName...` and `interfaceName`